ExperimentManager1.py

```diff
@@ -187,14 +204,19 @@ def StateActivity(self, state, mouse1simulated, mouse2simulated):
                 self.reward_manager.deliver_reward('dd', 2, self.punishment_time)
             """
         elif state == States.WaitForReturn:
-            self.sound_manager.play_sound('beep')
+            #self.sound_manager.play_sound('beep')            # Micky: Do we need a beep hear?
+            Play(Sounds.Start)
 
         elif state == States.TrialCompleted:
             # Increment the trial number counter
             self.numcompletedtrial += 1
             print("Trial Completed. Number of completed trials: ", self.numcompletedtrial)
             # self.trial_logger.log_trial_data(self.numcompletedtrial, "Completed Trial", self.opponent_choice, self.mouse_choice, self.mouse_reward, self.opponent_reward)
-
+                                                        # Micky: Why are the logger writes, commented out? There is no need to.
```

Comment:
have corrected in latest version

Reply:
OK. I am keeping the comment for the time being.